addOns/replacer/src/main/java/org/zaproxy/zap/extension/replacer/ReplacerParam.java

```diff
@@ -51,6 +51,9 @@ public class ReplacerParam extends AbstractParam {
     private static final String CONFIRM_REMOVE_RULE_KEY = REPLACER_BASE_KEY + ".confirmRemoveToken";
     private static final String FALSE_STRING = "false";
 
+    protected static final String REPORT_TO_REGEX = "(?i)report-(?:to|uri)";
+    protected static final String REPORT_TO_REPLACEMENT = "report-disabled";
```

Comment:
I couldn't think of a better way to do this and make it test friendly, the default patterns aren't exposed and clear or set requires the config to be saved which doesn't work for tests.